api/src/controllers/kots/KotsAPI.ts

```diff
@@ -391,7 +393,7 @@ export class KotsAPI {
         }
       }
     } catch {
-      /* no need to handle, rbac problem or not a path we can read registry */
+      // no need to handle, rbac problem or not a path we can read registry
```

Comment:
generally not in favor of superfluous refactoring, but this gets a ❤️

Reply:
i find linting errors very helpful and sometimes can catch bugs. when the file is full of lint issues its easy to ignore them.